e2e/tests/ui_tests/damages/1v2CreateClaim_DiffSol_test.js

```diff
@@ -29,7 +29,7 @@ Feature('1v2 Different Solicitors Claim Journey @e2e-unspec @e2e-nightly @e2e-un
 
 Scenario('Claimant solicitor raises a claim against 2 defendants who have different solicitors', async ({I}) => {
   await I.login(config.applicantSolicitorUser);
-  await I.createCase(claimant1, null, respondent1, respondent2);
+  await I.createCase(claimant1, null, respondent1, respondent2, 20000);
```

Comment:
It would probably be better to have a variable instead of a hardcoded number.

Reply:
I can define a constant, but I dont see a point as this is the value being sent as parameter